packages/api/src/hooks/useAuthorize.ts

```diff
@@ -11,10 +11,9 @@ const useAuthorize = () => {
     const current_token = getActiveAuthTokenIDFromLocalStorage();
     const invalidate = useInvalidateQuery();
     const { switchEnvironment } = useAPIContext();
-
     const { data, ...rest } = useQuery('authorize', {
         payload: { authorize: current_token || '' },
-        options: { enabled: Boolean(current_token) },
+        options: { enabled: Boolean(current_token), staleTime: 1000 * 60 },
```

Comment:
Whats the actual reason for multiple authorise calls? I have feeling that this is hiding the actual problems elsewhere.

Reply:
We use `useActiveWallet`, `useActiveAccount` that use `useAuthorize` everywhere. And in every place it calls it i supposed

Reply:
sounds like architecture/data flow problem, ideally, we should improve data flow (e.g. one possible strategy is to just move data/hooks higher in components hierarchy and pass them down to child components; if not viable, then context to provide global stuff;)